### DISCUSSION TITLE
Patreon / sponsorship support?

### DISCUSSION THREAD

**Discussion by jeromebaum** [agzam/spacehammer#133] [Q&A] [answered]
I'd like to support spacehammer via Patreon or some other sort of sponsorship (open collective, etc.)

Do you have something like this?

**Comment by agzam** [comment, marked as answer]
@jeromebaum Aww, that's very nice of you. From the bottom of my heart - thank you, it means a world to me! 

I have never bothered about setting up a sponsorship because this project was created to fulfill my personal needs. I never thought it would become something people would be interested in using. 

Besides that, unfortunately, my busy life makes it challenging to find time to work on Spacehammer as much as I wanted to. Frankly, I'm also extremely disorganized :). A recent couple of months were especially difficult for me; I lost my job due to the COVID-19 crisis. Fortunately, I was able to find a new one (it took much longer than usual), and my first day is tomorrow. For the next few months, I'm afraid I won't be able to focus on this project almost at all. Setting up a sponsorship (regardless of the amount of contributions) implies certain commitments that I cannot make right now. 

However, this project is no longer driven only by me, @eccentric-j  also has done and still does a lot of work, and he may have something to say about this.

Also, don't forget - sometimes throwing some ideas and writing issues makes even better support. 

I can only promise you this - I am still very much using Spacehammer, I have all sorts of ideas that I'm hoping we will be able to implement someday soon.

**Comment by jaidetree** [comment]
@jeromebaum Thank you. That's really a really kind offer.

As @agzam said we mostly just work on it in our free time between projects or as some immediate need arises. It's far from a full-time commitment at this point and the number of issues has been manageable so far.

There's room for improvement and ideas to explore but I'm not sure more money is going to make that happen much faster. Additionally, the nature of this project is to encourage people to make their own customizations and evolve it to fit their needs. 

At some point, I think a Patreon will be a great idea. Not necessarily for Agzam and I specifically, but say for incentivizing the creators of fennel and jeejah to improve the REPL experience and related editor plugins.

We are humbled by your willingness to support this project and sincerely thank you for giving it a shot and considering us. Perhaps sometime in the future, we will set something up, but as of now, I don't feel we are ready.

If you are concerned about this project being abandoned the good news is that it's hard to ignore this project seeing as both Agzam and I use it every day.

Until then, if you (or any future readers) are motivated to support this project please consider any of the following:

- Creating issues, discussions, or feature requests
- Create PRs for new features you feel benefit a wide user base
- Share it with others
- Help people with issues
- Provide feedback on new releases or when people create feature requests\discussions
- Even emoji reactions are a good way to let us know when people are excited, what they like, and what they dislike
